From e30d4578974b2cd1395e78811d5dad693f267c3f Mon Sep 17 00:00:00 2001
From: Ben Menchaca <ben.menchaca@qca.qualcomm.com>
Date: Tue, 11 Jun 2013 10:50:08 -0500
Subject: [netdev] inline kfree_skbmem

Our call to kfree_skbmem in the consume_skb change does not inline
at the optimization levels we usually use for our compiler.  This gives
the required hint to inline it.

Signed-off-by: Ben Menchaca <ben.menchaca@qca.qualcomm.com>
---
 net/core/skbuff.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-3.3.8/net/core/skbuff.c
===================================================================
--- linux-3.3.8.orig/net/core/skbuff.c	2013-08-01 12:57:06.709791013 -0500
+++ linux-3.3.8/net/core/skbuff.c	2013-08-01 12:57:06.709791013 -0500
@@ -562,7 +562,7 @@
 /*
  *	Free an skbuff by memory without cleaning the state.
  */
-static void kfree_skbmem(struct sk_buff *skb)
+static inline void kfree_skbmem(struct sk_buff *skb)
 {
 	struct sk_buff *other;
 	atomic_t *fclone_ref;
